### PR TITLE
Add browser and extension version to uninstall survey

### PR DESCRIPTION
**Pull request by andrey-p**
**Reviewer:** @jdorweiler 

**CC:** @alohaas 
**Dependencies:** internal PR

## Description:

Adds some more info to the uninstall survey.

## Steps to test this PR:

1. Uninstall the extension
2. You should see the browser version and the extension version in the initial request to `atb.js`.

## Automated tests:
- [ ] Unit tests
- [ ] Integration tests

###### Reviewer Checklist:
- [ ] **Ensure the PR solves the problem**
- [ ] **Review every line of code**
- [ ] **Ensure the PR does no harm by testing the changes thoroughly**
- [ ] **Get help if you're uncomfortable with any of the above!**
- [ ] Determine if there are any quick wins that improve the implementation

###### PR Author Checklist:
- [ ] Get advice or leverage existing code
- [ ] Agree on technical approach with reviewer (if the changes are nuanced)
- [ ] Ensure that there is a testing strategy (and documented non-automated tests)
- [ ] Ensure there is a documented monitoring strategy (if necessary)
- [ ] Consider systems implications

### CODE REVIEW

File: shared/js/utils.js [hunk 1]
```diff
@@ -29,10 +29,10 @@ require.scopes.utils = ( () => {
 
     function parseUserAgentString (uaString) {
         if (!uaString) uaString = window.navigator.userAgent
-        const rgx = uaString.match(/(Firefox|Chrome|Safari)\/([0-9]+)/)
+        const rgx = uaString.match(/(Firefox|Chrome|Safari)\/([0-9.]+)/)
         return {
             browser: rgx[1],
-            majorVersion: rgx[2]
+            version: rgx[2]
```
Comment (andrey-p) [inline shared/js/utils.js]: I changed it to full version when I updated the feedback info, changing this for consistency.

File: shared/js/atb.js [hunk 2]
```diff
@@ -155,7 +155,16 @@ var ATB = (() => {
...
+
+            let browserInfo = utils.parseUserAgentString()
+            let browserName = browserInfo.browser
+            let browserVersion = browserInfo.version
+            let extensionVersion = window.chrome.runtime.getManifest().version
+
+            if (browserName) url += `&browser=${browserName}`
+            if (browserVersion) url += `&bv=${browserVersion}`
+            if (extensionVersion) url += `&v=${extensionVersion}`
+
```
Comment (andrey-p) [inline shared/js/atb.js]: This will probably lead to a conflict with my other PR, annoyingly.

**Comment by alohaas** [comment]
👍

**Comment by jdorweiler** [comment]
👍

**Pull request merged**